files/angular-base-apps/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-base-apps",
+  "repo": "base-apps/angular-base-apps",
+  "files": {
+    "include": ["base-apps.css", "base-apps.min.css", "base-apps.js", "base-apps.min.js", "base-apps-templates.js", "base-apps-templates.min.js"]
```

Comment:
This way you are including the files from the / directory. https://github.com/base-apps/angular-base-apps
Which doesnt have them. You need to set the correct paths relative do your file structure in your repo.